tests/integration/projects/cpp_make_subdir_edge_cases_project/src/Makefile

```
@@ -0,0 +1,24 @@
+	$(CXX) $(CXXFLAGS) -c main.cpp -o main.o
+
+models/base.o: models/base.cpp ../include/models/base.hpp
+	$(CXX) $(CXXFLAGS) -c models/base.cpp -o models/base.o
+
+models/entities.o: models/entities.cpp ../include/models/entities.hpp ../include/models/base.hpp
+	$(CXX) $(CXXFLAGS) -c models/entities.cpp -o models/entities.o
+
+services/processor.o: services/processor.cpp ../include/services/processor.hpp ../include/models/base.hpp
+	$(CXX) $(CXXFLAGS) -c services/processor.cpp -o services/processor.o
+
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)